src/pinns_pde_solve.jl

```diff
@@ -31,11 +31,7 @@ function PhysicsInformedNN(dx,
                            strategy = GridTraining(),
                            kwargs...)
     if init_params === nothing
-        if chain isa FastChain
-            initθ = DiffEqFlux.initial_params(chain)
-        else
-            initθ,re  = Flux.destructure(chain)
-        end
+        initθ = DiffEqFlux.initial_params(chain)
```

Comment:
is there a test that says this is okay with Flux.Chain? I would think no?

Reply:
fastchain = FastChain(FastDense(2,16,Flux.σ),FastDense(16,16,Flux.σ),FastDense(16,1))
chain = Chain(Dense(2,16,Flux.σ),Dense(16,16,Flux.σ),Dense(16,1))

initθ1 = DiffEqFlux.initial_params(chain)
initθ2,_  = Flux.destructure(chain)

initθ1 == initθ2

Reply:
it is the wrapper : `initial_params(f::Chain) = Flux.destructure(f)[1]` 
https://github.com/SciML/DiffEqFlux.jl/blob/b642d5c917b33e2593165a2e1b076f4080eb6a9b/src/fast_layers.jl#L5